app/controllers/api/articles_controller.rb

```diff
@@ -0,0 +1,34 @@
+class Api::ArticlesController < ApplicationController
+  def index
+    if params[:article_type]
+      articles_list = find_article_type(params[:article_type])
+
+      render json: {
+        articles: articles_list
+      }
+    else
+      render json: {
+        articles: Article.all
+      }
```

Comment:
Is this else block really a use case since you will never send all articles? 🤔

Reply:
Potentially not!